pkg/proxy/ipvs/graceful_termination.go

```diff
@@ -167,7 +167,8 @@ func (m *GracefulTerminationManager) deleteRsFunc(rsToDelete *listItem) (bool, e
 			// For UDP traffic, no graceful termination, we immediately delete the RS
 			//     (existing connections will be deleted on the next packet because sysctlExpireNoDestConn=1)
 			// For other protocols, don't delete until all connections have expired)
-			if strings.ToUpper(rsToDelete.VirtualServer.Protocol) != "UDP" && rs.ActiveConn+rs.InactiveConn != 0 {
+			proto := strings.ToUpper(rsToDelete.VirtualServer.Protocol)
+			if proto != "UDP" && proto != "SCTP" && rs.ActiveConn+rs.InactiveConn != 0 {
```

Comment:
should this use `conntrack.IsClearConntrackNeeded`? it's not quite the same thing, but...

either way, the comment above is wrong now